Modelica/Thermal/HeatTransfer/UsersGuide/References.mo

```diff
@@ -0,0 +1,24 @@
+within Modelica.Thermal.HeatTransfer.UsersGuide;
+class References "References"
+  extends Modelica.Icons.References;
+  annotation (preferredView="info",Documentation(info="<html>
+
+<table border=\"0\" cellspacing=\"0\" cellpadding=\"2\">
+
+  <tr>
+    <td>[Fischer1999]</td>
+    <td>R. Fischer, <em>Elektrische Maschinen</em> (in German), 10th edition, Hanser-Verlag, 1999</td>
+  </tr>
+
+  <tr>
+    <td>[Holman1997]</td>
+    <td>J. P. Holman, <em>Heat Transfer</em>, 8th edition, McGraw-Hill, 1997</td>
+  </tr>
```

Comment:
I could download the 10th edition from https://www.academia.edu/38166004/Heat_Transfer_10thEdition_by_JP_Holman.pdf after registration. Should we update and add the link?

Reply:
Yes I will upgrade to the 10th edition and add the link